src/components/SelectInput/index.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import {
+    _cs,
+    listToMap,
+} from '@togglecorp/fujs';
+
+import SelectInputContainer from '../SelectInputContainer';
+import { rankedSearchOnList } from '../../utils';
+
+import styles from './styles.css';
+
+type OptionKey = string | number;
```

Comment:
We should support boolean too.

Reply:
Should we though?

Reply:
Yes

Reply:
Or create a Toggle button and leave this be :D

Reply:
We have ToggleButton